src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -823,7 +823,10 @@ protected boolean mineBlock(BlockPos targetPos, BlockRailBase.EnumRailDirection
         EntityPlayerMP fakePlayer = CartTools.getFakePlayerWith(this, head);
 
         // Fires break event within; harvest handled separately
-        if (!WorldPlugin.playerRemoveBlock(world, targetPos, fakePlayer, false))
+        BlockEvent.BreakEvent breakEvent = new BlockEvent.BreakEvent(world, targetPos, targetState, CartTools.getCartOwnerEntity(this));
```

Comment:
Are we not using the fake player here?

Reply:
ah, missed that when i rollback the changes.